pages/common/snowsql.md

```diff
@@ -0,0 +1,24 @@
+# snowsql
+
+> SnowSQL command-line client for Snowflake's Data Cloud.
+> More information: <https://docs.snowflake.com/en/user-guide/snowsql.html>.
+
+- Connect to an instance at {{account}}.snowflakecomputing.com (password can be provided in prompt or configuration file):
+
+`snowsql --accountname {{account}} --username {{username}} --dbname {{database}} --schemaname {{schema}}`
+
+- Connect to an instance specified by the configuration file (by default, `~/.snowsql/config`):
+
+`snowsql --config {{.snowsql/config}}`
+
+- Connect to the default instance, specifying a MFA token:
+
+`snowsql --mfa-passcode {{token}}`
+
+- Execute a single SQL query or SnowSQL command on the default connection (useful in shell scripts):
+
+`snowsql --query '{{query}}'`
+
+- Execute commands from a file on the default connection:
```

Comment:
```suggestion
- Execute commands from a specific file on the default connection:
```


- This change is optional - no corresponding style guide requirement exist.
- The corresponding change should be applied to `.pl` translation if it's used here.
- We often prefer use `a specific` statement.
- There is no requirement to enclose referred to placeholders in braces in descriptions.